### ISSUE TITLE
Add Tink support for EU market

### ISSUE THREAD

**Issue by rltbg** [maybe-finance/maybe#198]
Problem Description:
I'm frustrated when trying to access financial data from various European banks and institutions because there are no options supported.

Solution Requested:

Tink (https://tink.com/)  is a European-based financial aggregation platform that offers a free tier for users. This platform supports a wide range of European banks and institutions, making it a compelling option for users in Europe. By integrating Tink into the financial services ecosystem, we can provide a robust solution for European users to access their financial data easily and securely.

Alternative Solutions Considered:
Integration of Woob (https://woob.tech/): Another solution is to explore the integration of Woob, an open-source project based in France. While Woob currently supports only French businesses, there is potential for the community to expand its capabilities to include more banks. By contributing to and supporting the development of Woob, we can work towards a more inclusive financial aggregation solution for All users.

Additional Context:
Currently, many financial aggregation services focus on American banks and institutions, leaving European users underserved. This limitation creates frustration and inconvenience for users who require access to their European financial data. It is essential to explore and implement solutions like Tink and Woob to bridge this gap and provide European users with a comprehensive financial aggregation experience.

**Comment by noonfyi** [comment]
Yes, support for institutions outside of the US has been demonstrated and I agree that it should be in the plans. There's discussion around non-US support in a couple locations, but to gather several in one spot here:

#60 :
> When we built Maybe we offered financial advice and as such had a LOT more regulatory stuff to account for. So, that just meant we couldn't support non-US folks. Adding an issue now to remove that limitation. The problem will still be that we have to add data sources for folks outside North America, but otherwise you should still be able to actually create an account and manually add stuff.

#64 (closed in #73):
> There are some checks during signup/onboarding where it asks for location and then if you don't select United States it shows you a message about not being supported. We need to remove that so there's no "location" limitation.

@Shpigford [on Discord](https://discord.com/channels/1191501936281784361/1197328450042732584/1197333517491765392):
> absolutely plan on supporting all countries. we just recently made the repo public after we shut down the original product (which was US-only). so, got a lot of foundational stuff to do before we start getting it functioning well for non-US markets.

---

For Tink specifically, a free account appears to be limited to using sandbox data and not live data and I haven't been able to find any documentation otherwise related to free limits. Though I'd be happy to be wrong about this!
![image](https://github.com/maybe-finance/maybe/assets/116238122/b26f0548-c19b-4542-bbfc-4706b032fe75)
Screenshot from the Overview console of a free account.
